src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -387,27 +389,20 @@ private void testPluginAgainst(
         // potential javadoc execution to avoid general test failure.
         if (!ranCompile) {
             runner.run(
-                    mconfig,
+                    Map.of("maven.javadoc.skip", "true"),
                     pluginCheckoutDir,
                     buildLogFile,
                     "clean",
-                    "process-test-classes",
-                    "-Dmaven.javadoc.skip");
+                    "process-test-classes");
         }
         ranCompile = true;
 
         List<String> args = new ArrayList<>();
-        Map<String, String> userProperties = mconfig.userProperties;
-        args.add(
-                String.format(
-                        "--define=forkCount=%s", userProperties.getOrDefault("forkCount", "1")));
```

Comment:
Getting ourselves out of the business of `forkCount`: this is now delegated to consumers (both of which were already passing in a value).